app.js

```diff
@@ -0,0 +1,367 @@
+import KEY_BUTTONS from './src/keyConfig.js';
+import Button from './src/button.js';
+import AltButton from './src/altButton.js';
+import LetterButton from './src/letterButton.js';
+
+class Keyboard {
+  constructor(textarea) {
```

Comment:
You added ability to connect any textarea to your virtual keyboard. Cool!

Reply:
Thank you!